tests/testthat/test_pipelines.R

```diff
@@ -151,3 +151,40 @@ test_that("survtoclassif_disctime", {
   # model with more covariates should have better C-index
   expect_gt(pred2$score(), pred$score())
 })
+
+skip_if_not_installed("mlr3extralearners")
+
+test_that("survtoclassif_IPCW", {
+  requireNamespace("mlr3extralearners")
+
+  task = tsk("rats")
+  split = partition(task)
+  task_train = task$clone()$filter(split$train)
+  task_test = task$clone()$filter(split$test)
+
+  pipe = mlr3pipelines::ppl("survtoclassif_IPCW", learner = lrn("classif.gam"),
+                            cutoff_time = 50)
+  expect_class(pipe, "Graph")
+
+  grlrn = mlr3pipelines::ppl("survtoclassif_IPCW", learner = lrn("classif.gam"),
+                             cutoff_time = 50, graph_learner = TRUE)
+  expect_class(grlrn, "GraphLearner")
+  suppressWarnings(grlrn$train(task_train))
```

Comment:
we get an error, see [here](https://stackoverflow.com/questions/12953045/warning-non-integer-successes-in-a-binomial-glm-survey-packages) for a solution? or change learner?

Reply:
`lrn("classif.rpart")` is the easiest to do as is available in `mlr3` and supports `weights`